views/event/index.hbs

```diff
@@ -8,16 +8,16 @@
 		<a class='btn btn-light ms-lg-3 mt-3 mt-lg-0 d-block d-lg-inline' href='new'>+ Add Event</a></p>
 		{{/if}}
 
-		<div class='row mb-3 row-cols-1 row-cols-md-2 d-none'>
+		<div class='row mb-3 row-cols-1 row-cols-md-2'>
 			<div class='col mb-3'>
-				<div class='form-group text-start'>
-					<label class='form-label' for='year'>Year</label>
-					<select id='year' class='form-control'>
-						{{#each years}}
-						<option value='{{this}}' {{#eq this ../year }} selected{{/eq}}>{{this}}</option>
-						{{/each}}
-					</select>
-				</div>
+				<div class="form-group text-start">
+				<label class="form-label" for="year">Season</label>
+				<select id="year" class="form-control" onchange="location.href=`?season=${this.options[selectedIndex].value}`">
+					{{#each seasons}}
+					<option value="{{this.id}}" {{#eq this.id ../season}} selected{{/eq}}>{{this.Identifier}}</option>
+					{{/each}}
+				</select>
+			</div>
 			</div>
 
 			<div class='col mb-3'>
```

Comment:
By removing the `d-none`, you've also made the search box visible (which currently does nothing!)

I'd probably either add `d-none` to this `col` or consider deleting it entirely